lib/app/models/testing_instance.rb

```diff
@@ -182,14 +180,15 @@ def patient_id=(patient_id)
         return if patient_id.to_s == self.patient_id.to_s
 
         resource_references.destroy
+        patient_ids = get_requirement_value('patient_ids')
 
         # For patient id list, don't clear it out but rather add it to the list of known
         # patients to pull from.
-        self.patient_ids = if patient_ids.blank?
-                             patient_id
-                           else
-                             patient_ids.split(',').append(patient_id).uniq.join(',')
-                           end
+        if patient_ids.blank?
+          set_requirement_value('patient_ids', patient_id)
```

Comment:
You should be able to use the getter/setter here.

Reply:
yep, fixed